### DISCUSSION TITLE
Create make enum command

### DISCUSSION THREAD

**Discussion by felipeArnold** [laravel/framework#50002] [Ideas] [answered]
Within the commands defined in the structure there is currently no option to create enums, whenever I create a new project I need to create a custom command to create enums, the same applies to external actions and services.

For example: when I need to create an action, I created a make:action command.
For an external service, for integrations with other products I generally use a make:service command.

In this case, I always need to create a new command to create enums, I believe that make:enum would be very useful for Laravel developers' daily lives.

Att. Felipe Arnold.

**Comment by hamzaelmaghari** [comment]
As many don't use enums in their code bases, I think Laravel may not add tons of commands. Instead you could create a new console command `php artisan make:command EnumCommand` then create `enum.stub` in your stubs directory which will be injected from your `EnumCommand` class.

NB: Laravel  introduced create class command, you may extend that 

Regards,

**Comment by henze-housepedia** [comment]
I don't. I use IDE for that. And even if I need Laravel specific classes, I have IDE for that with Laravel IDEA...

**Comment by crynobone** [comment, marked as answer]
This will be available in Laravel 11.
